src/redux/categories/categories.js

```diff
@@ -0,0 +1,32 @@
+const CHECK_STATUS = 'bookstore/books/CHECK_STATUS';
```

Comment:
- [ ] Kindly notice that the check status action should be like this `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.